specification/web/resource-manager/Microsoft.Web/stable/2021-03-01/KubeEnvironments.json

```diff
@@ -575,6 +645,10 @@
               "$ref": "#/definitions/AppLogsConfiguration",
               "description": "Cluster configuration which enables the log daemon to export\napp logs to a destination. Currently only \"log-analytics\" is\nsupported"
             },
+            "containerAppsConfiguration": {
```

Comment:
In the current CLI custom sdk, this property was added to "KubeEnvironments" instead of "KubeEnvironmentPatchResource"

Reply:
@sanchitmehta , where should this property be?